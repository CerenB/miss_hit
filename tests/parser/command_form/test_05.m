% (c) Copyright 2019 Florian Schanda

function test_05()
  cd ../foo
end
