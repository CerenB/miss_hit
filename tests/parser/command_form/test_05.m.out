----------------------------------------------------------------------
--  Parse tree for test_05.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_05>
    Body: Sequence_Of_Statements
      Statements: Naked_Expression_Statement
        Expression: Command form Function_Call
          Name: Identifier <cd>
          Arguments: Char_Array_Literal '../foo'
----------------------------------------------------------------------
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
